usersimcrs/run_simulation.py

```diff
@@ -283,16 +286,30 @@ def load_rasa_diet_classifier(
     agent_intents_str = intent_schema["agent_elicit_intents"]
     agent_intents_str.extend(intent_schema["agent_set_retrieval"])
     agent_intents = [Intent(intent) for intent in agent_intents_str]
-    nlu = IntentClassifierRasa(
+    intent_classifier = IntentClassifierRasa(
         agent_intents,
         config["rasa_dialogues"].get(),
         ".rasa",
     )
-    nlu.train_model()
-    return nlu
+    intent_classifier.train_model()
+    return intent_classifier
 
 
 if __name__ == "__main__":
     args = parse_args()
     config = load_config(args)
-    main(config)
+
+    # Defines the agent for the simulation.
+    # By default, a local moviebot is used. See TODO for more details.
+    moviebot_uri = config["moviebot_uri"].get()
+    try:
+        response = requests.get(moviebot_uri)
+        assert response.status_code == 200
+        agent = MovieBotAgent(agent_id="MovieBotTester", uri=moviebot_uri)
+    except requests.exceptions.RequestException:
+        raise RuntimeError(
+            f"Connection refused to {moviebot_uri}. Please check that "
+            "moviebot is running at this address. See the full traceback above."
```

Comment:
replace `moviebot_uri` with `agent_uri`. In the exception text, "Please check that the conversational agent is ..."